### DISCUSSION TITLE
Everyone on earth wants less "For You" algorithmic bullshit, not more

### DISCUSSION THREAD

**Discussion by twome** [community/community#63361] [Other Feature Feedback, Questions, & Ideas]
Yet more examples of the ways in which capitalism is fundamentally opposed to democracies

**Comment by darkwater4213** [comment]
Disagree. Just disable the whole "For You" tab by default.

By the way, political editorials, correct or not (I happen to disagree on both grounds) are tremendously unhelpful and unproductive on a _programming site_. Look at my aside. Does it add any value to our discussion? Probably not, aside from satirical value. I could go off on how you're misrepresenting democracy, capitalism, and our governmental system (assuming you happen to live in the U.S., as I do). But that wouldn't be productive, would it? What is more productive is this whole rebuke, which also happens to function as a sort of meta-satire. How convenient.

**Reply by twome** [reply]
Wow you write *exactly* like every other programmer who incorrectly assumes that because they can think their way around an algorithm, they're automatically "smart" enough to know anything whatsoever about politics, history or power. Go whiff your own farts somewhere else or start [learning the 101s](https://www.marxists.org/archive/marx/works/1867-c1/ch01.htm#S1)

**Reply by T800-101A** [reply]
I guess now Git will come to be another "Social Media" like "the others"?
I'm New at this, but really into it and I love it . 
GitHub just make it easier to be focus in things that are related with my future, my career. 
I just realize that no one speak politics here... and that's cool.
I have a very strong opinion, experiences and knowledge about Politics... so much that I love Politics...too.
To awake people and change the world , political sense, you should have common...sense. 
Keep it Nice and easy, smile and fix the problem if you don't like it. 
bye

**Reply by twome** [reply]
Wow you write exactly like every other programmer who incorrectly assumes that because they can think their way around an algorithm, they're automatically "smart" enough to know anything whatsoever about politics, history or power. Start [learning the 101s](https://www.marxists.org/archive/marx/works/1867-c1/ch01.htm#S1)

**Reply by T800-101A** [reply]
Actually I'm more than a "Programmer". We do not own this World, but we do live here. Peace and Respect.

**Reply by twome** [reply]
Sorry @T800-101A - I was referring to @darkwater4213, not you!

**Reply by darkwater4213** [reply]
Of course, you have no idea how knowledgeable I may or may not happen to be in those fields in addition to programming. As it happens, I was into philosophy and politics long before I was into programming. I have found that the rigorous thinking that programming requires has helped me immensely in other intellectual pursuits, but that's neither here nor there. Why? Because my response wasn't political _in the slightest_.

**Reply by twome** [reply]
@darkwater4213 No, you exposed plenty of flaws in your knowledge in your first response, cheers!

**Reply by darkwater4213** [reply]
> @darkwater4213 No, you exposed plenty of flaws in your knowledge in your first response, cheers!

I fail to see how. Care to enlighten me?

**Reply by darkwater4213** [reply]
> @darkwater4213 No, you exposed plenty of flaws in your knowledge in your first response, cheers!

I fail to see how. Care to enlighten me?

**Reply by twome** [reply]
@darkwater4213 Read Capital then we can talk on even footing

**Reply by darkwater4213** [reply]
> @darkwater4213 Read Capital then we can talk on even footing

How do you mean? I don't know what book that is off the top of my head.

**Reply by twome** [reply]
@darkwater4213 [yes, that's exactly the problem](https://www.marxists.org/archive/marx/works/1867-c1/ch01.htm#S1)

**Reply by darkwater4213** [reply]
This is Cave Johnson. We're done here.

**Reply by twome** [reply]
Please stop coming back here to leave useless comments if you're not going to do basic reading on the subject

**Comment by adrianorodriguesbr** [comment]
❤️

**Comment by SeanCassiere** [comment]
While I do fundamentally agree with you on this algorithmic BS on apps with Twitter, YouTube, Facebook, etc...
I've found the "For You" on GitHub to be quite nice for finding repos at an early stage of their development. It's been giving me relevant and interesting repos to check out.
My sentiment on this would definitely change should this recommendation engine get hijacked for paid promotions.

**Reply by twome** [reply]
Yeah I was being overly general for effect. I like algorithmic suggestions when they're 100% opt-in and don't replace proper user-customisable summary / feed UIs - which is unfortunately never the case for profit-motivated social media companies squeezing users into greater platform engagement and dependence. YouTube's suggestions used to be really good, aeons ago, and I hear Spotify's are still pretty effective.

**Comment by ericoporto** [comment]
Disable everything so the feed is empty and use this one instead

https://github.com/dashboard-feed

**Comment by Aiolos13** [comment]
Nice project

**Comment by Game77ok** [comment]
Githug

**Comment by Game77ok** [comment]
> Githug

**Comment by GllmR** [comment]
![image](https://github.com/community/community/assets/56537238/ce85e481-4e9d-4987-a30b-1be5a6cbdd19)

That new feed sucks...
